include/h2o.h

```diff
@@ -134,6 +134,10 @@ typedef struct st_h2o_handler_t {
      * `req->entity` already contains a complete request body by checking if `req->proceed_req` is NULL.
      */
     unsigned supports_request_streaming : 1;
+    /**
+     *
+     */
+    unsigned supports_expect : 1;
```

Comment:
```suggestion
    unsigned handles_expect : 1;
```
Maybe a matter of tastes, but I think "handles" makes it clear how the handler "supports" expect.

Reply:
Renamed in https://github.com/h2o/h2o/pull/3419/commits/5fcfd4dcb9694c3584e1bcb0cdfae52f7c3cacab